### ISSUE TITLE
Output in non-verbose unattended installation

### ISSUE THREAD

**Issue by ThomDietrich** [openhab/openhabian#810]
Check where these come from...

```
2020-03-25_21:09:05_UTC [openHABian] Executing 'openhabian-setup.sh unattended'...
2020-03-25_21:09:05_UTC [openHABian] Checking for root privileges... OK
Hit:1 http://raspbian.raspberrypi.org/raspbian stretch InRelease
Hit:2 http://archive.raspberrypi.org/debian stretch InRelease
Reading package lists...
2020-03-25_21:09:27_UTC [openHABian] Loading configuration file '/etc/openhabian.conf'... OK

```

**Comment by ThomDietrich** [comment]
Did someone miss to redirect the output of https://github.com/openhab/openhabian/blob/3bfe531b92265e585186b956c011ce736643f7e5/openhabian-setup.sh#L62
?

Solution? `cond_redirect apt-get update`

**Comment by mstormi** [comment]
I added a fix for this to PR #724
